mmv1/third_party/terraform/resources/resource_storage_bucket.go.erb

```diff
@@ -248,6 +248,22 @@ func resourceStorageBucket() *schema.Resource {
 				Description: `The bucket's Versioning configuration.`,
 			},
 
+			"autoclass": {
```

Comment:
Can this field be updated? If so, would you mind adding an update step for the test as well? Otherwise, please add `forcenew` to the object and the subfield. Thanks!